.github/workflows/windows.yml

```diff
@@ -38,7 +38,7 @@ jobs:
             elixir-version: '1.14.5'
             os: 'windows-latest'
             version-type: 'strict'
-          - otp-version: '25'
+          - otp-version: '25.1'
```

Comment:
`25.2.5.2` (identified by `25` has an install issue). Maybe this should be reported next to Erlang/OTP. I searched their CI and don't know where the install is tested in GitHub CI (e.g. `windows-2022`).